account_payment_order/models/account_move.py

```diff
@@ -103,22 +101,19 @@ def create_account_payment_line(self):
                 if new_payorder:
                     move.message_post(
                         body=_(
-                            "%d payment lines added to the new draft payment "
-                            "order %s which has been automatically created."
+                            "%(count)d payment lines added to the new draft payment "
+                            "order %(payorder.name)s which has been automatically created."
```

Comment:
@wlin-kencove do you know if this syntax works?

Reply:
I am more used to seeing the parameterized % or f(string) formatting. This method should work as well, though it is usually accompanied by a dictionary that maps the values. 

I am expecting something that looks like the example below:
```
body = _(%(count)d payment lines added to the new draft payment "
                "order %(payorder.name)s which has been automatically created." % 
               {
                   "count": count,
                   "payorder.name": payorder.name
               }

) 

```

Reply:
I could be mistaken if this is a new way of formatting strings, but they do look similar (except for the missing dictionary).

I do see it implemented in a later code segment:
![image](https://user-images.githubusercontent.com/59095052/139849704-f37c6a3b-3263-45f9-b617-22650053e1bf.png)

So perhaps missed  in this section?